homeassistant/components/media_player/plex.py

```diff
@@ -347,6 +347,7 @@ def refresh(self, device, session):
         if device:
             self._device = device
             self._session = None
+            self._media_title = None
```

Comment:
Some notes:
1) We should clear out all of the `self._media*` properties if we know nothing is playing.
2) Can we move this all to one location? I'm picturing something like this at the end of refresh:
```python
if not self._is_player_active:
    self._media_title = None
    self._media_image_url = None
    …
```

Also, this is a monster of a method and could use some refactoring, but that doesn't need to be part of this PR.

Reply:
A quick test and commenting out my line under "device" and doing your not active resulted in the issue resurfacing.

Going redo my PR with linter fix and if you could approve that i'll look into refactoring the refresh in another PR

Reply:
Strange. Well in any case we need to treat all of the media info properties the same. You might want to create a helper method to clear them all out and then call it in these locations.

Reply:
Am I able to do that under another PR that way I can do a refactor on that refresh call and hopefully better understand it.